package.json

```diff
@@ -69,7 +69,7 @@
     "minimist": "0.0.8",
     "mkdirp": "0.3.2",
     "mongodb": "1.4.14",
-    "nib": "0.5.0",
+    "nib": "1.1.0",
```

Comment:
Old version of `nib` throws exception for `linear-gradient(-180deg,#3c3c57 11%,#8598be 100%)` since it uses degree as argument. Newer version has a fix of this bug